src/cca/scripts/cloud_socks_proxy_server.ts

```diff
@@ -1,15 +1,34 @@
+import * as cloud_social_provider from '../../lib/cloud/social/provider';
 import * as social from '../../interfaces/social';
 import * as uproxy_core_api from '../../interfaces/uproxy_core_api';
+import CoreConnector from '../../generic_ui/scripts/core_connector'
 
 import { SocksProxy } from '../model/socks_proxy_server';
 
+export function MakeCloudSocksProxy(corePromise: Promise<CoreConnector>,
```

Comment:
Since it just returns synchronously, it doesn't seem like this helper function gives you much over the constructor?